integration-cli/docker_api_images_test.go

```diff
@@ -206,3 +206,33 @@ func (s *DockerSuite) TestAPIImagesSizeCompatibility(c *check.C) {
 		c.Assert(image.Size, checker.Not(checker.Equals), int64(-1))
 	}
 }
+
+// Regression : #38171
+func (s *DockerSuite) TestAPIImagesFilterMultiReferenceMatch(c *check.C) {
```

Comment:
Ideally, those need to be in the `integration` package (`integration-cli` is deprecated — see #32866). That said it's weird that the CI is green (aka this means it doesn't validate correctly this anymore…)

Reply:
Thanks for remind me, update.